pkg/monitor/controller.go

```diff
@@ -365,3 +369,38 @@ func (monitor *controllerMonitor) deleteAgentCRD(name string) error {
 	}
 	return nil
 }
+
+func (monitor *controllerMonitor) antreaAgentInfoAPIAvailable(stopCh <-chan struct{}) bool {
+	groupVersion := v1beta1.SchemeGroupVersion.String()
+	checkFunc := func() (done bool, err error) {
+		resources, err := monitor.client.Discovery().ServerResourcesForGroupVersion(groupVersion)
+		if err != nil {
+			if !errors.IsNotFound(err) {
+				return false, err
+			}
+			klog.InfoS("No server resources found for GroupVersion", "groupVersion", groupVersion)
+			return false, nil
+		}
+		for _, resource := range resources.APIResources {
+			if resource.Kind == agentInfoResourceKind {
+				return true, nil
+			}
+		}
+		return false, nil
+	}
+
+	found := false
+	if err := wait.PollImmediateUntil(time.Minute, func() (done bool, err error) {
+		var checkErr error
+		found, checkErr = checkFunc()
+		if checkErr != nil {
+			klog.ErrorS(err, "Error getting server resources for GroupVersion, will retry after 1m", "groupVersion", groupVersion)
+			return false, nil
+		}
+		return true, nil
+	}, stopCh); err != nil {
+		klog.ErrorS(err, "Failed to get Server resources for GroupVersion", "groupVersion", groupVersion)
+		found = false
```

Comment:
```suggestion
		klog.ErrorS(err, "Failed to get server resources for GroupVersion", "groupVersion", groupVersion)
```
to keep consistent logs

Reply:
updated.